data/strings/strings.txt

```diff
@@ -18655,32 +18655,45 @@
 
   [bookmarks_import]
     tags = android
-    comment = Bookmark categories screen
-    en = Import bookmarks
-    ar = إضافة الإشارات المرجعية من ملفات الجهاز
-    be = Імпартаваць закладкі
-    bg = Импортиране на отметки
-    ca = Importa marcadors
-    cs = Importovat záložky
-    de = Lesezeichen importieren
-    es = Importar marcadores
-    et = Impordi järjehoidjad
-    eu = Inportatu markagailuak
-    fi = Tuo kirjamerkit
-    fr = Importer des signets
-    hu = Könyvjelzők importálása
-    it = Importa luoghi preferiti
-    lt = Importtuoti adresyną
-    mr = खूणपत्रे आयात करा
-    nb = Importer bokmerker
-    nl = Bladwijzers importeren
-    pl = Import zakładek
-    pt = Importar favoritos
-    pt-BR = Importar favoritos
-    ro = Importă locuri preferate
-    ru = Импортировать метки
-    tr = Yer imlerini içe aktar
-    zh-Hant = 匯入書籤
+    comment = Bookmark categories screen, button that opens folder selection dialog to import KML/KMZ/GPX/KMB files
+    en = Import Bookmarks and Tracks
+    ar = استيراد الإشارات المرجعية والمسارات
+    be = Імпартаваць закладкі і cцежкі
+    bg = Импортиране на отметки и песни
+    ca = Importa marcadors i les pistes
+    cs = Import záložek a stop
+    da = Importer bogmærker og spor
+    de = Lesezeichen und tracks importieren
```

Comment:
```suggestion
    de = Lesezeichen und Tracks importieren
```

Reply:
tracks are called 'Strecken' in other parts of the UI. This should be kept consistent. 
I'm unsure whether 'Tracks' or 'Strecken' is better. Maybe 'GPS Tracks'?

Reply:
I agree that it should be kept consistent.
And imho "Strecken" does not sound good.

I would prefer to either use "Tracks" or "Routen", what do you think?!

E.g Komoot calls their tracks "Routen", from what I saw.

Reply:
I don't like "Strecken" as well.
I think "Tracks" is best, as it's the most commonly used word.

Reply:
I would suggest to stick with `Tracks` here.

A separate PR should be created to change all other occurrences of "Strecke(n)" to "Track(s)", which I can do later today.

Reply:
Created PR: https://github.com/organicmaps/organicmaps/pull/5387